spec.md

```diff
@@ -582,6 +604,12 @@ message ControllerPublishVolumeRequest {
   // Whether to publish the volume in readonly mode. This field is
   // REQUIRED.
   bool readonly = 5;
+
+  // OPTIONAL field for storing principal information to be used in
+  // the publishing of this volume. If the principal is present the
+  // plugin MAY ensure the volume published will be accessible for
+  // the herein specified entities only.
+  Principal container_acts_as = 6;
```

Comment:
What's the obligation of the CO w/ respect to passing the same `container_acts_as` value to both `ControllerVolumePublish` and `NodeVolumePublish` RPCs? For example, must they be identical, or is it possible that the principal of the latter call is a subset of the principal of the former call? How would a CO understand the limitations here?